pocket_coffea/utils/plot_utils.py

```diff
@@ -259,10 +259,13 @@ def plot_data_mc_hist1D(h, histname, config):
             )
             ax.set_ylim((0, 1.20 * max(stack_sum(stack_mc_nominal).values())))
             rax.set_ylim((0.8, 1.2))
-            if config.variables[histname].axes[0].lim != (0,0):
+            if config.variables[histname].axes[0].lim != (0, 0):
                 ax.set_xlim(*config.variables[histname].axes[0].lim)
             else:
-                ax.set_xlim(config.variables[histname].axes[0].start, config.variables[histname].axes[0].stop)
+                ax.set_xlim(
+                    config.variables[histname].axes[0].start,
+                    config.variables[histname].axes[0],
```

Comment:
Why did the `.stop` at the end of this line disappeared?

Reply:
this is probably an unintended edit, sorry about that